experimental/ragged_inference_v2/ragged_inference_v2/garbage_pad_ragged_acts.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from typing import List
+
+import numpy as np
+import torch
+import triton
+import triton.language as tl
+
+
+@triton.jit
+def add_kernel(
```

Comment:
ok to remove this ? I can update the branch, spring cleanup so that it's easier to build upon down the line

Reply:
done